OpenFeature/src/main/java/dev/openfeature/sdk/events/EventHandler.kt

```diff
@@ -12,21 +11,18 @@ import kotlinx.coroutines.flow.filterIsInstance
 import kotlinx.coroutines.launch
 import kotlin.reflect.KClass
 
-interface ProviderStatus {
-    fun isProviderReady(): Boolean
-}
-
-interface EventObserver {
+interface ProviderEventObserver {
     fun <T : OpenFeatureEvents> observe(kClass: KClass<T>): Flow<T>
+    fun isProviderReady(): Boolean
 }
```

Comment:
Nit picking: I'd prefer to keep the same naming for observer/publisher: `EventObserver` and `EventPublisher` seems a bit cleaner. `isProviderReady` is also not really part of an Observer pattern? I wonder if the previous interface (before this PR) was better. But this is also not a blocking comment on my side 👍

Reply:
makes sense. I try to keep it as it was.